src/main/java/appeng/tile/AEBaseTile.java

```diff
@@ -280,52 +281,36 @@ public NBTTagCompound getUpdateTag()
 	public void handleUpdateTag( NBTTagCompound tag )
 	{
 		final ByteBuf stream = Unpooled.copiedBuffer( tag.getByteArray( "X" ) );
-		if( this.readFromStream( stream ) )
+
+		if( this.readUpdateData( stream ) )
 		{
 			this.markForUpdate();
 		}
 	}
 
-	private final boolean readFromStream( final ByteBuf data )
+	protected boolean readFromStream( final ByteBuf data ) throws IOException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)